components/bookshelf/Book.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react"
+import Image from "next/image"
+import {
+  BookCard,
+  BookContent,
+  BookImage,
+  HeaderBar,
+  InfoBox,
+  Author,
+  Publisher,
+  Title,
+  Synopsis,
+  SideBar,
+  ExtendedSideBar,
+} from "../styled/bookshelf"
+
+const Book = ({ book }) => {
```

Comment:
can you add an `IBook` or `BookType` that will have all the attributes that you'll get from `book`?

you can put on a separate shared file with `export`. Not sure which one SP prefers for `IBook` vs `BookType`

Sample below (Sorry if attrs are incorrect)

```
interface IBook {
 name: string;
 author: string;
 rating: number;
}

interface BookProps {
 book: IBook;
}

const Book = ({ book }: BookProps) => {
```

Reply:
![image](https://user-images.githubusercontent.com/91857581/188588559-f47a8e03-da56-4d57-be9a-6d47f1793137.png)